components/builder-api/src/server/resources/profile.rs

```diff
@@ -210,7 +211,7 @@ fn update_account(req: HttpRequest,
                   body: Json<UserUpdateReq>,
                   state: Data<AppState>)
                   -> HttpResponse {
-    let account_id = match authorize_session(&req, None) {
+    let account_id = match authorize_session(&req, None, RoleAction::Read, None) {
```

Comment:
Should this be `Write`?

Reply:
No in this case, a user needs to be able to update their own account. If we alter this to `::Write` they would be unable to do so if they have the role of 'member'